src/style.css

```diff
@@ -0,0 +1,140 @@
+body {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  background: #e5e5e5;
+}
+
+.conteiner {
+  width: 200px;
+  height: 408px;
+}
+
+.Product {
+  height: 406px;
+  width: 198px;
+  background: #fff;
+}
+
+img {
+  width: 160px;
+  height: 134px;
+  margin: 33px 20px 0;
+}
+
+h1 {
+  width: 166px;
+  height: 36px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+  margin: 40px 16px 4px;
+}
+
+h2 {
+  width: 98px;
+  height: 14px;
+  color: #616070;
+  margin: 0 84px 0 16px;
+  font-weight: 400;
+  font-size: 10px;
+}
+
+.Product__price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 16px;
+  width: 166px;
+  height: 18px;
+}
+
+span {
+  height: 18px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
```

Comment:
Move all repeatable colors to the variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)